documentation/adapter-implementation.md

```diff
@@ -1,20 +1,18 @@
-# Implementing an X-Road adapter using XRd4J
+# Implementing an X-Road adapter using XRd4J <!-- omit in toc -->
 
-## Table of Contents
+## Table of Contents <!-- omit in toc -->
```

Comment:
`<!-- omit in toc -->` that's a very interesting feature. 
Thank you so much. I learned something new today :)